assets/pages/bundles-landing/components/Bundle.jsx

```diff
@@ -0,0 +1,48 @@
+// ----- Imports ----- //
+
+import React from 'react';
+
+import DoubleHeading from 'shared-components/doubleHeading';
+import FeatureList from 'shared-components/featureList';
+import InfoText from 'shared-components/infoText';
+import CtaLink from 'shared-components/ctaLink';
+
+
+// ----- Component ----- //
+
+export default function Bundle(props) {
+
+    return (
+      <div className="bundles__bundle">
+        <DoubleHeading
+          heading={props.heading}
+          subheading={props.subheading}
+        />
+        <FeatureList listItems={props.listItems} />
+        {props.infoText ? <InfoText text={props.infoText} /> : ''}
+        <CtaLink text={props.ctaText} url={props.ctaLink} />
+      </div>
+    );
+
+}
+
+
+// ----- Proptypes ----- //
+
+Bundle.defaultProps = {
+    subheading: '',
+    listItems: [],
+    infoText: '',
+};
+
+Bundle.propTypes = {
+    heading: React.PropTypes.string.isRequired,
```

Comment:
According to React documentation, React. [PropTypes](https://facebook.github.io/react/docs/typechecking-with-proptypes.html) is deprecated. We should use [prop-types](https://www.npmjs.com/package/prop-types) library if we want to achieve something like this. Alternatively, Could we use [Flow + React](https://flow.org/en/docs/frameworks/react/) instead?